engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -1094,6 +1094,11 @@ namespace dmGraphics
 
         delete[] device_list;
 
+        if (created_width != context->m_Width || created_height != context->m_Height)
+        {
+            dmPlatform::SetWindowSize(context->m_Window, created_width, created_height);
+        }
```

Comment:
If the swapchain has been created at a different resolution we need to make sure the GLFW layer knows about the new updated resoultion.